Company policy network troubleshooting

jdiehl opened the pull request:
Explain ways to connect to Make It Native in a closed network

Commit:
Company policy network troubleshooting

Explain ways to connect to Make It Native in a closed network

ConnorLand commented:
I'll update Support with the following message after this PR is merged:

"Hello all. 

Some improvements have been made early regarding port forwarding documentation. Specifically, if someone is at a company with very restricted WiFi, it can be difficult for them to connect their Make It Native app to their project in Studio Pro and then sync changes/develop live.

This doc update gives customers 3 remedies to these issues. If you have a place in the support knowledge base for port forwarding/MIN troubleshooting, please add this doc to that list: https://github.com/mendix/docs/pull/5294/files"

ConnorLand committed:
Change formatting

In docs, we used numbered lists to convey sequential instructions (First do this, then this, then this). Therefore I changed the numbered list to an unordered list.
